plasma_framework/contracts/src/exits/payment/PaymentInFlightExitable.sol

```diff
@@ -0,0 +1,317 @@
+pragma solidity ^0.5.0;
+pragma experimental ABIEncoderV2;
+
+import "./PaymentExitDataModel.sol";
+import "./spendingConditions/IPaymentSpendingCondition.sol";
+import "./spendingConditions/PaymentSpendingConditionRegistry.sol";
+import "../utils/ExitId.sol";
+import "../utils/ExitableTimestamp.sol";
+import "../utils/OutputId.sol";
+import "../../framework/PlasmaFramework.sol";
+import "../../framework/interfaces/IExitProcessor.sol";
+import "../../utils/Bits.sol";
+import "../../utils/IsDeposit.sol";
+import "../../utils/SliceUtils.sol";
+import "../../utils/OnlyWithValue.sol";
+import "../../utils/UtxoPosLib.sol";
+import "../../utils/Merkle.sol";
+import "../../transactions/PaymentTransactionModel.sol";
+import "../../transactions/outputs/PaymentOutputModel.sol";
+
+contract PaymentInFlightExitable is
+    IExitProcessor,
+    OnlyWithValue,
+    PaymentSpendingConditionRegistry
+{
+    using ExitableTimestamp for ExitableTimestamp.Calculator;
+    using IsDeposit for IsDeposit.Predicate;
+    using PaymentOutputModel for PaymentOutputModel.Output;
+    using RLP for bytes;
+    using RLP for RLP.RLPItem;
+    using UtxoPosLib for UtxoPosLib.UtxoPos;
+
+    uint constant public MAX_INPUT_NUM = 4;
+    uint256 public constant IN_FLIGHT_EXIT_BOND = 31415926535 wei;
+    mapping (uint192 => PaymentExitDataModel.InFlightExit) public inFlightExits;
+
+    PlasmaFramework private framework;
+    IsDeposit.Predicate private isDeposit;
+    ExitableTimestamp.Calculator private exitableTimestampCalculator;
+
+    event InFlightExitStarted(
+        address indexed owner,
```

Comment:
Probably we should consistent with the `RootChain.sol`, which is calling `initiator` instead of `owner`. And I think it makes sense as it doesn't need to be `owner` of any output to start exit.

```
address indexed initiator,
```

Reply:
right, done